osmaxx-py/osmaxx/excerptexport/views.py

```diff
@@ -29,9 +29,9 @@
 class NewExtractionOrderView(LoginRequiredMixin, FrontendAccessRequiredMixin, View):
     def get(self, request, excerpt_form_initial_data=None):
         active_excerpts = Excerpt.objects.filter(is_active=True)
-        active_bbox_excerpts = active_excerpts.filter(bounding_geometry__bboxboundinggeometry__isnull=False)
+        active_bbox_excerpts = active_excerpts.filter(_bounding_geometry__bboxboundinggeometry__isnull=False)
```

Comment:
We're using a private 'variable' to filter by? If we should use this attribute, should it really be marked as _don't touch_/internal?

[Pep 8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)
[PEP 8 Stackoverflow Explanation](http://stackoverflow.com/questions/1641219/does-python-have-private-variables-in-classes)